src/legacy/ui/public/vislib/visualizations/_labels.scss

```diff
@@ -0,0 +1,22 @@
+$visBarLabelDarkColor: $euiColorDarkShade;
+$visBarLabelLightColor: $euiColorLightShade;
```

Comment:
@mmeshi You are right about the light vs dark shade colors. In EUI, we actually use full white and black for these types of patterns. So you can use:

```
$visColumnChartBarLabelDarkColor: $euiColorGhost;
$visColumnChartBarLabelLightColor: #000; // EUI doesn't yet have a variable for fully black in all themes
```

Reply:
Also, please move this file to be next to the JS file where the classes are applied.